README.md

```diff
@@ -28,13 +28,11 @@ socket.query({
 
 #### `var socket = dns([options])`
 
-Create a new DNS socket instance. Options include:
+Create a new DNS socket instance. The `options` object includes:
 
-``` js
-{
-  socket: customDgramSocket
-}
-```
+- `retries` <Number>: Number of total query attempts made during `timeout`. Default: 5.
+- `socket` <Object>: A custom dgram socket. Default: A `'udp4'` socket.
+- `timeout` <Number>: Total timeout in milliseconds after which a `'timeout'` event is emitted. Default: 7500.
```

Comment:
for some reason the rendered markdown looks weird with this (all the next sections are indented). any ideas why this happens?